catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -120,6 +141,38 @@ public String toString() {
         .orElse(Collections.emptyList());
   }
 
+  @Override
+  public List<QueryMetacardImpl> getQueryMetacards(WorkspaceMetacardImpl workspaceMetacard) {
+    List<String> queryIds =
+        getAssociationMetacards(workspaceMetacard.getId())
+            .stream()
+            .map(AssociationMetacardImpl::new)
+            .map(AssociationMetacardImpl::getAssociationTarget)
+            .collect(Collectors.toList());
+
+    Query query = new QueryImpl(getQueriesFilter(queryIds));
+    QueryRequest queryRequest = new QueryRequestImpl(query);
```

Comment:
:pencil2: `QueryRequest queryRequest = new QueryRequestImpl(new QueryImpl(getQueriesFilter(queryIds)));`

Reply:
I prefer to keep it this way to reduce nesting